### ISSUE TITLE
Fetched author info is not used

### ISSUE THREAD

**Issue by muan** [koddsson/coworking-with#12]
#10 added fetching github users, which is shown when as responses to the coworking command but `.git/hooks/commit-msg` still only checks for `git log`.. unless I'm missing something?

```
$ cowork dgraham
Coauthor 'dgraham' was not found in git log. Trying to fetch info from GitHub..
Found the user in GitHub!
Coauthor 'dgraham' will be attributed as 'David Graham <...@gmail.com>'.
Happy coworking!
```

```
$ cat .git/hooks/commit-msg
#!/bin/bash

commitMessage=$1

function getCoauthor {
  signiture=$(git log --no-merges -1 --author "$1" --format='%an <%ae>' 2>/dev/null)
  echo "Co-authored-by: $signiture"
}
...
```

```
 $ cat .git/config 
...
...
[coworking]
	coauthor = dgraham
```

**Comment by koddsson** [comment]
Yup! I experienced this recently but just haven't had time to fix it. I got some time now to take a look.

**Comment by koddsson** [comment]
Released in [v0.3.1](https://github.com/koddsson/coworking-with/releases/tag/v0.3.1).
